bigtable-hbase-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/read/BigtableResultScannerAdapter.java

```diff
@@ -50,15 +53,19 @@ public BigtableResultScannerAdapter(ResponseAdapter<T, Result> rowAdapter) {
    * @return a {@link org.apache.hadoop.hbase.client.ResultScanner} object.
    */
   public ResultScanner adapt(
-      final com.google.cloud.bigtable.grpc.scanner.ResultScanner<T> bigtableResultScanner) {
+      final com.google.cloud.bigtable.grpc.scanner.ResultScanner<T> bigtableResultScanner,
+      final Span span) {
```

Comment:
Wouldn't make more sense to push this down into RetryingReadRowsOperation so that we don't have to duplicate this for callback api?

Reply:
This span is HBase specific as a wrapper for a Span created by RetryingReadRowsOperation.